deployment/script/build-CCNP.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -o errexit
+
+ccnp_url=https://github.com/intel/confidential-cloud-native-primitives.git
+device_plugin=docker.io/library/ccnp-device-plugin:0.1
+get_quote=docker.io/library/ccnp-quote-server:0.1
+measurement=docker.io/library/ccnp_measurement_server:0.1
+eventlog=ccnp_eventlog_server:0.1
+feature_url=https://github.com/kubernetes-sigs/node-feature-discovery.git
```

Comment:
NDF 0.14 is relesed, pls try to use the image from docker hub directly:
https://github.com/kubernetes-sigs/node-feature-discovery/releases/tag/v0.14.2